packages/@glimmer/destroyable/index.ts

```diff
@@ -191,6 +191,12 @@ export function destroyChildren(destroyable: Destroyable) {
   iterate(children, destroy);
 }
 
+export function hasDestroyableChildren(destroyable: Destroyable) {
```

Comment:
Maybe underscore? I'm vaguely worried that folks in glimmer.js land might have access to it directly...